src/calendar/reducers.js

```diff
@@ -52,6 +52,18 @@ function events(state = initialEvents, action) {
                     },
                 },
             })
+        case "DELETE_EVENT_SUCCESS":
+            return update(state, {
+                [action.event.date.getFullYear()]: {
+                    [action.event.date.getMonth()]: {
+                        $splice: [
+                            [state[action.event.date.getFullYear()][action.event.date.getMonth()].findIndex(event => (
+                                event == action.event
```

Comment:
This does work, but should compare events by ObjectId instead of by reference. (Also, if the comparison were to be by reference, this could just be indexOf.)